src/hooks/use-computation-notification-count.ts

```diff
@@ -6,32 +6,86 @@
  */
 import { useSelector } from 'react-redux';
 import { ReduxState } from '../redux/reducer.type';
+import ComputingType from 'components/computing-status/computing-type';
+import RunningStatus from 'components/utils/running-status';
+import { isNodeBuilt } from 'components/graph/util/model-functions';
 
+/**
+ * Custom hook that calculates the number of computation notifications.
+ * It checks the status of various computations and determines if a notification should be displayed.
+ * @returns The count of computation notifications.
```

Comment:
notification is sooooo misnamed....
useComputationResultsCount ?
the number of computation results accessible ?

Reply:
done